plugins/source/okta/client/spec.go

```diff
@@ -1,6 +1,60 @@
 package client
 
-type Spec struct {
-	Token  string `json:"token,omitempty"`
-	Domain string `json:"domain"`
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+type (
+	Spec struct {
+		Token     string     `json:"token,omitempty"`
+		Domain    string     `json:"domain,omitempty"`
+		RateLimit *RateLimit `json:"rate_limit,omitempty"`
+	}
+	RateLimit struct {
+		MaxBackoff time.Duration `json:"max_backoff,omitempty"`
+		MaxRetries int32         `json:"max_retries,omitempty"`
+	}
+)
+
+const (
+	OktaAPIToken = "OKTA_API_TOKEN"
+)
+
+func (s *Spec) setDefaults() {
+	const (
+		minRetries = int32(3)
+		minBackOff = 5 * time.Second
```

Comment:
Looks like the default is 30s with 2 retries (based on a quick google finding this: https://github.com/okta/okta-sdk-golang/blob/master/okta/okta.go#L187); if users were already running into errors before we should probably keep the default backOff to 30s or so as well (the minimum can still be less)